src/main/java/com/blockscore/models/Address.java

```diff
@@ -183,4 +191,6 @@ public String getPostalCode() {
   public String getCountryCode() {
     return countryCode;
   }
+
 }
+
```

Comment:
@dgollahon  without `@jsonProperty` Address object cannot be created from json response while searching watch list. Empty objects will be created for address.